src/main/java/net/minecraftforge/event/anvil/AnvilDamageEvent.java

```diff
@@ -0,0 +1,87 @@
+package net.minecraftforge.event.anvil;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.world.World;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+@Cancelable
+public class AnvilDamageEvent extends Event
+{
+    /**
+     * The current state of the Anvil pre-damage
+     */
```

Comment:
This documentation should be on the getters, not the fields. The getters are the public API.